typescript/packages/noosphere-guide/get-started.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: layouts/docs.njk
+---
+
+# Get started with Noosphere
+
+Noosphere supports a few different programming environments, including:
+
+- [Rust][noosphere-rust-docs]
+- <a target="_blank">TypeScript (web, via NPM)</a>
+- <a target="_blank">Swift</a>
+- <a target="_blank">C</a>
```

Comment:
```suggestion
- TypeScript (web, via NPM)
- Swift
- C
```

don't make it look like a link if it's not a link? We can add it later.

Reply:
Check the preview. I have a CSS-based thing for this.